test/e2e/kubevirt.go

```diff
@@ -1474,24 +1572,21 @@ chpasswd: { expire: False }
 `
 
 			userDataWithIperfServer = userData + fmt.Sprintf(`
-packages:
-  - iperf3
```

Comment:
I assume this is now present in the new test img you're pulling ?

Reply:
Yep, now that we have a proper fedora 39 test tooling image (we full IPv6 support) we can use it, so we don't have to install iperf3 ourselves.